Auth0Templates/LogIn.html

```diff
@@ -268,6 +268,11 @@
         color: #b10e1e;
     }
 
+    .govuk-link {
+        font-family: "Source Sans Pro", "HelveticaNeue",
+            "Helvetica Neue", "Arial", "Helvetica", sans-serif !important;
+    }
+
```

Comment:
Not sure why we need to define font-family at the component level. It should be at the global settings and declared at the style.css level.